### DISCUSSION TITLE
Package Microsoft.AspNetCore.OData 8.2.5 and OData 8 Previews

### DISCUSSION THREAD

**Discussion by christophwille** [OData/AspNetCoreOData#1238] [General]
https://devblogs.microsoft.com/odata/odata-net-8-preview-release/

Does the 8.2.5 release play nicely with those previews or would we need something newer? (We are using AspNetCore.OData today, and "naked" OData 8 Preview seems a steep upgrade)

Thanks.

**Comment by habbes** [comment]
Hello @christophwille, due to some of the breaking changes in OData Core 8 preview, it's not compatible with Microsoft.AspNetCore.OData 8. It would require some changes to Microsoft.AspNetCore.OData to adapt to the public API changes. Since those API changes are also exposed to consumers of AspNetCore.OData, then we'd have to publish a new major version of Microsoft.AspNetCore.OData. We'll release a preview version that's compatible with OData.NET 8 soon.

**Reply by christophwille** [reply]
Just wanted to complete this discussion by posting https://devblogs.microsoft.com/odata/microsoft-aspnetcore-odata-9-preview-release/
